lib/bloom/sbbf.ts

```diff
@@ -221,7 +221,7 @@ class SplitBlockBloomFilter {
     private desiredFalsePositiveRate: number = SplitBlockBloomFilter.DEFAULT_FALSE_POSITIVE_RATE
     private numBlocks: number = 0
     private numDistinctValues: number = SplitBlockBloomFilter.DEFAULT_DISTINCT_VALUES
-    private hashStrategy = new parquet_thrift.BloomFilterHash(parquet_thrift.XxHash)
+    private hashStrategy = new parquet_thrift.BloomFilterHash(new parquet_thrift.XxHash())
```

Comment:
@shannonwells @enddynayn This is the one line that was odd for me.

Reply:
That is weird. Tou shouldn't have needed to do this

Reply:
parquet_thrift.XxHash is referring to a class. Perhaps we are using the wrong thing?

Reply:
Looks like it is just the way that the thrift struct is translated for typescript. It all works either way.